src/DashboardHub/Bundle/AppBundle/Service/TravisService.php

```diff
@@ -47,9 +48,34 @@ public function getBuilds($reponame, $max = 5)
                 ,
                 true
             );
-            $cache->set($builds, 600);
+            $cache->set($builds, 60);
```

Comment:
I was going to do it in this branch, but you know what, lets get it out there & I will create a Ticket to do it & assign it to me.